requirements.txt

```diff
@@ -90,7 +90,7 @@ rsa==3.4.2
 ruamel.yaml==0.15.96
 s3transfer==0.1.13
 sensu-plugin==0.3.1
-service-configuration-lib==2.4.3
+-e git+https://github.com/Yelp/service_configuration_lib@930b4c79f523ec29554447f6ba709f6869ac371f#egg=service-configuration-lib # TODO: replace once this change is released
```

Comment:
TODO: use `service-configuration-lib==2.4.4` when https://github.com/Yelp/service_configuration_lib/pull/48 is released